LICENSE.md

```diff
@@ -0,0 +1 @@
+TBD
```

Comment:
How strange. I thought I had added the NASA license indicated in the NSIDC GitHub template when I created this repo. That's the license we should be using according to the template notes.

Reply:
Followup: I have a `LICENSE` file in my workspace (no `.md`) with the NASA license content.

Reply:
🤦🏽 Sorry I don't know how I missed that

Reply:
The deleted `LICENSE` file contained the NASA license we need to include in this project. Need to move the deleted content into this `.md` version.